rabbus.go

```diff
@@ -349,6 +339,32 @@ func (ri *RabbusInterpreter) declareExchange(ex, kind string) error {
 	return ri.ch.ExchangeDeclare(ex, kind, ri.config.Durable, false, false, false, nil)
 }
 
+func (ri *RabbusInterpreter) closeConnAndChan() (err error) {
+	if err = ri.ch.Close(); err != nil {
+		return
```

Comment:
Missing `return err` 🤔

Reply:
We've got declared on the return signature `(err error)` in this case we can have empty `returns`